sspl_test/coverage/coverage_code

```diff
@@ -1,9 +1,9 @@
 from coverage import Coverage
 co = Coverage(
-            data_file='/tmp/sspl/.sspl_report',
+            data_file='/var/cortx/sspl/coverage/.sspl_coverage_report',
```

Comment:
pickup data dir path from sspl_constants

Reply:
currently, we are calling `start_sspl_coverage.sh` `stop_sspl_coverage.sh` scripts from run_qa_test.sh
this is why this is hardcoded text since I am extracting it in a shell script as it is,
if we call python script  `coverage_setup.py`  with args (which is currently unused),
then we can introduce variables like DATA_PATH in this python script and eliminate hard-coded text.

Reply:
>if we call python script coverage_setup.py with args

After we discussed, am ok with this